packages/app/src/components/SourceForm.tsx

```diff
@@ -233,6 +233,16 @@ export function LogTableModelForm({
             connectionId={connectionId}
           />
         </FormRow>
+        <FormRow label={'Body Expression'}>
```

Comment:
expose `bodyExpression` since we need to extract k8s event details from the json body field